aspnetcore/choose-aspnet-framework.md

```diff
@@ -38,6 +38,7 @@ ASP.NET is a mature framework that provides all the services needed to build ent
 
 ## ASP.NET Core scenarios
 
+* [Razor Pages](xref:mvc/razor-pages/index) is the recommended approach to create a Web UI with ASP.NET Core 2.0.
```

Comment:
If you look at how "scenarios" is used here, Razor Pages doesn't fit -- it's atechnology for implementing a scenario; if Razor Pages fit here, Web Pages and MVC would fit under old ASP.NET.

It fits better in the "which one is right" table where we have 

Use MVC or Web API

for Core.